DFC.App.DiscoverSkillsCareers.Services/Api/AssessmentApiService.cs

```diff
@@ -0,0 +1,109 @@
+﻿using DFC.App.DiscoverSkillsCareers.Models.Assessment;
+using DFC.App.DiscoverSkillsCareers.Services.Contracts;
+using System.Net.Http;
+using System.Net.Mime;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DFC.App.DiscoverSkillsCareers.Services.Api
+{
+    public class AssessmentApiService : IAssessmentApiService
+    {
+        private readonly HttpClient httpClient;
+        private readonly ISerialiser serialiser;
+        private readonly IDataProcessor<GetQuestionResponse> getQuestionResponseDataProcessor;
+        private readonly IDataProcessor<GetAssessmentResponse> getAssessmentResponseDataProcessor;
+
+        public AssessmentApiService(
+            HttpClient httpClient,
+            ISerialiser serialiser,
+            IDataProcessor<GetQuestionResponse> getQuestionResponseDataProcessor,
+            IDataProcessor<GetAssessmentResponse> getAssessmentResponseDataProcessor)
+        {
+            this.httpClient = httpClient;
+            this.serialiser = serialiser;
+            this.getQuestionResponseDataProcessor = getQuestionResponseDataProcessor;
+            this.getAssessmentResponseDataProcessor = getAssessmentResponseDataProcessor;
+        }
+
+        public async Task<NewSessionResponse> NewSession(string assessmentType)
+        {
+            var url = $"{httpClient.BaseAddress}/assessment?assessmentType={assessmentType}";
```

Comment:
Instead of repeating $"{httpClient.BaseAddress}, wouldn't of been cleaner to create a class level variable and re-use that.